looprpc/server.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package looprpc;
 
+option go_package = "github.com/lightninglabs/loop/looprpc";
```

Comment:
Why do we need to add this, out of curiosity?

Reply:
I think this is mandatory in the new version of protoc we use. But in general it's good practice to set.